sample/example/example.en.custom_word_breaker/source/example.en.custom_word_breaker.model.ts

```diff
@@ -1,4 +1,5 @@
-/// <reference types="@keymanapp/lexical-model-types" />
+// Defines LexicalModelSource.
+/// <reference types="@keymanapp/lexical-model-compiler/dist/kmlmc.d.ts" />
```

Comment:
This allows VS Code Intellisense to link to the `LexicalModelSource` type.  Granted, I have a feeling that it's probably relocated a bit for 17.0... but I think the compiler almost always ignores the definition or lack thereof for the type.  It's mostly there for Intellisense when someone actually looks at the example.